service/telemetry/telemetry.go

```diff
@@ -147,6 +154,22 @@ func newLogger(cfg LogsConfig, options []zap.Option) (*zap.Logger, error) {
 		zapCfg.EncoderConfig.EncodeTime = zapcore.ISO8601TimeEncoder
 	}
 
+	if cfg.Rotation != nil && cfg.Rotation.Enabled {
+		rotationSchema := "rotation-" + uuid.NewString()
```

Comment:
Do we need randomness here? Why can't we just register a reasonably unique schema, e.g. `otelcol-rotation`

Reply:
As there will never be several loggers with different rotation config, I think `otelcol-rotation` is OK.